make/migrations/postgresql/0030_2.0.0_schema.up.sql

```diff
@@ -173,3 +173,14 @@ CREATE TABLE artifact_2
   push_time     timestamp,
   CONSTRAINT unique_artifact_2 UNIQUE (project_id, repo, tag)
 );
+
+CREATE TABLE audit_log
+(
+ id             SERIAL PRIMARY KEY NOT NULL,
+ project_id     int NOT NULL,
+ operation      varchar(20) NOT NULL,
+ resource_type  varchar(255) NOT NULL,
+ resource       varchar(255) NOT NULL,
```

Comment:
The length of repository+digest may exceeds  255

Reply:
extend it to 1024